core/src/main/java/hudson/triggers/SCMTrigger.java

```diff
@@ -178,7 +208,7 @@ public File getLogFile() {
         return new File(job.getRootDir(),"scm-polling.log");
     }
 
-    @Extension @Symbol("scm")
+    @Extension @Symbol("pollScm")
```

Comment:
It does. As @jglick mentions elsewhere, we could change this to `@Symbol({"pollScm", "scm"})` but I don't think anyone's actually using the `scm` symbol currently other than Pipeline users...and it doesn't *work* in Pipeline. So. =)